src/sonic-config-engine/minigraph.py

```diff
@@ -1364,6 +1364,11 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
     for port in ports.values():
         port['pfc_asym'] = 'off'
 
+    # make admin status to down if port not parsed from minigraph
+    for port in ports.keys():
+        if 'admin_status' in ports[port] and port not in port_speeds_default:
```

Comment:
This check if `'admin_status' in ports[port]`. What would be the value if its present? Where is that default value coming from?